crates/orchestrator/src/data_storage/aws_s3/mod.rs

```diff
@@ -48,7 +48,15 @@ impl DataStorage for AWSS3 {
     async fn get_data(&self, key: &str) -> Result<Bytes> {
         let response = self.client.get_object().bucket(&self.bucket).key(key).send().await?;
         let data_stream = response.body.collect().await.expect("Failed to convert body into AggregatedBytes.");
+        tracing::debug!("DataStorage: Collected response body into data stream from {}, key={}", self.bucket, key);
         let data_bytes = data_stream.into_bytes();
+        tracing::info!(
```

Comment:
missed this info log?

Reply:
yeah, converted it to a debug for both put and set